tool/extract_deps.py

```diff
@@ -400,25 +398,29 @@ def run_commands_in_sequence(commands, initial_input=None):
                 "-Dverbose",
                 "-DoutputType=json",
                 "-DoutputFile=/dev/stdout",
-                "-f",
-                temp_pom_path,
             ],
-            ["grep", "-v", "\\[INFO"],
-            ["jq", "[.children | .. | {version, groupId, artifactId}?] | unique"],
+            ["grep", "-v", "^\\[\\(INFO\\|WARNING\\)"],
+            # Below: In some cases, for parent packages which need to access child pom files, the above returns several JSON objects
+            # This requires it to be "slurped" into a single JSON array, as the next command expects a single JSON object
+            ["jq", "--slurp", "[.[] | .children[]?] | unique | map({version, groupId, artifactId})"],
         ],
         "plugins": [  # Plugin dependencies
-            ["mvn", "dependency:resolve-plugins", "-f", temp_pom_path],
+            ["mvn", "dependency:resolve-plugins"],
```

Comment:
It is better to fix a version of `maven-dependency-plugin.`

Reply:
Is this also independent of `build status`? @randomicecube @LogFlames

Reply:
mvn dependency version fixed in #57